WindowsFunctionsStacks.json

```diff
@@ -52,6 +52,37 @@
                 "isDeprecated": null
             }
         },
+        {
+            "id": null,
+            "name": ".Net Framework",
+            "type": "Microsoft.Web/availableStacks?osTypeSelected=WindowsFunctions",
+            "properties": {
+                "name": ".Net Framework",
+                "display": ".Net Framework",
+                "dependency": null,
+                "majorVersions": [
+                    {
+                        "displayVersion": "4.7",
+                        "runtimeVersion": "4.7",
+                        "supportedFunctionsExtensionVersions": [
+                            "~1"
+                        ],
+                        "isDefault": true,
+                        "minorVersions": [],
+                        "applicationInsights": true,
+                        "appSettingsDictionary": {},
+                        "siteConfigPropertiesDictionary": {
+                            "Use32BitWorkerProcess": true
+                        },
+                        "isPreview": false,
+                        "isDeprecated": false,
+                        "isHidden": true
```

Comment:
We should be using is deprecated and not is hidden. Is hidden can be shown by preview flag. Is deprecated is only shown if the user is already running it

Reply:
Creating V1 should be allowed from CLIs/ VS Code but should be hidden in portal. And Function V1 is not deprecated yet. Does setting `isDeprecated` to `true` impact viewing existing V1 function apps in portal?

Reply:
In that case it should not be hidden either, it's current

Reply:
Portal will automatically hide in create since it's not v3. In configuration we want to expose this as the runtime for users on v1. Hidden is for something that we want to hide behind a feature flag

Reply:
So which field does Ux rely to not display .Net Framework? Does Ux filter options for ~3 only?

Reply:
Ok. I got the anwer. Thanks! Will udpate.